assets/pages/contributions-landing/contributionsLandingMetadata.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
```

Comment:
I sort of feel that if you're going to the trouble of moving this out into its own file, it would be a good opportunity to just use the existing utilities - but

Reply:
ah, maybe it's not as simple as I thought - but the combination of `currency.js` and `countryGroup.js` can at least provide the currency stuff that's duplicated here. Though maybe there's no good existing mechanism for varying copy between countries in a centralised place.

Don't worry about it for now!